### PR TITLE
Extending Wheel Data by Friction Coefficient

### PR DESCRIPTION
**Pull request by ThomasNaderBMW**
#### Add a description
Extending the Wheel Data transmitted by friction coefficients.
Doing this, it is possible to run a dynamic/vehicle model by using SensorView or out of standard -so far- TrafficUpdate.

**Some questions to ask**:
Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
It is a feature and not compatibility breaking.
How has it been tested?
Yes and it is the only missing value so far.

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [x] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html).
- [x] I have taken care about the [documentation](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/commenting.html).
- [x] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [x] My changes generate no errors when passing CI tests. 
- [ ] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

If you can’t check all of them, please explain why.
If all boxes are checked or commented and you have achieved at least one positive review, you can assign the label ReadyForCCBReview!

### CODE REVIEW

**Review by ndunningbmw — APPROVED** [review]
LGTM

**Review by ndunningbmw — APPROVED** [review]

**Review by HendrikAmelunxen — COMMENTED** [review]
Makes sense, but the reference should be adapted according the guideline
https://opensimulationinterface.github.io/osi-documentation/#_guidelines_for_contributions (4.3.11)

**Comment by ThomasNaderBMW** [comment]
Thank you for the hint @HendrikAmelunxen . Is it correct now?

File: osi_object.proto [hunk 1]
```diff
@@ -675,6 +675,17 @@ message MovingObject
             // 3d models.
             //
             optional string model_reference = 9;
+
+            // The value describes the kinetic friction of the tyre's contact point.
+            // If different friction coefficients due to more than one contact points are available,
+            // this value contains the average.
+            //
+            // Unit: Dimensionless.
```
Comment (stefancyliax) [inline osi_object.proto:683]: OSI CCB 23.05.2022.
Please remove stop.
```suggestion
            // Unit: Dimensionless
```

Comment (stefancyliax) [reply osi_object.proto:683]: @pmai to merge. Thank you!

**Review by pmai — APPROVED** [review]
As per CCB

**Pull request merged**